.github/workflows/main.yml

```diff
@@ -1,10 +1,8 @@
 name: .NET
 
 on:
-  push:
-    branches: [ master ]
-  pull_request:
```

Comment:
I still need these for the rest of the action flow

Reply:
Included it back again and added a condition to include github.event_name